### PR TITLE
update oraclelinux with smaller 8-slim image

### PR DESCRIPTION
**Pull request by mcalunod-olm**
[ARM64v8 8-slim] 2019-07-25-13

Signed-off-by: Michael Calunod <michael.calunod@oracle.com>

### CODE REVIEW

**Comment by Djelibeybi** [comment]
Ping @tianon and @yosifkit to review and merge ASAP. Our original `8-slim` image should've been called "8-anything-but-slim". This is the new significantly smaller and more deserving of the tag "slim" image.

**Comment by tianon** [comment]
This only updates `arm64v8` -- is that the only one that was `anything-but-slim`? :smile:

**Comment by Djelibeybi** [comment]
> This only updates `arm64v8` -- is that the only one that was `anything-but-slim`? 😄

The Arm one isn't slim but isn't used much either. We wanted to get this slimmer one out quickly and the Arm one will follow.

File: library/oraclelinux [hunk 1]
```diff
@@ -7,7 +7,7 @@ amd64-GitFetch: refs/heads/dist-amd64
 amd64-GitCommit: 9c633ff80efaaa9106a28eec3495cbc47cda534e
 # https://github.com/oracle/container-images/tree/dist-arm64v8
 arm64v8-GitFetch: refs/heads/dist-arm64v8
-arm64v8-GitCommit: 0e32e6e9a98f167d1da41a445a23ac33dedb0ff1
+arm64v8-GitCommit: 8e9e64fd5886bd4499e05399372caa7332d71589
```
Comment (yosifkit) [inline library/oraclelinux]: But arm is the only change here :arrow_up:. Is there an updated `amd64-GitCommit`?

Comment (Djelibeybi) [reply library/oraclelinux]: Yes, sorry. I got my bits backwards. I'm checking up internally now.

**Comment by yosifkit** [comment]
I'll merge this so the `arm64v8` updates can get building. Feel free to make another PR for any `amd64` updates.

**Pull request merged**